Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -27,7 +27,7 @@ private struct BrowserViewControllerUX {
 
 class BrowserViewController: UIViewController {
 
-    private var urlBar: URLBarView!
+    var urlBar: URLBarView!
```

Comment:
fixed -- yeah this is why i needed feedback haha i wasn't sure how to follow the coding paradigm, but now it's really clear to me, thanks!